pkg/reconciler/knativeserving/ingress/ingress.go

```diff
@@ -40,11 +40,17 @@ func ingressFilter(name string) mf.Predicate {
 	}
 }
 
+// noneFilter drops all ingresses but allows everything else.
+func noneFilter(u *unstructured.Unstructured) bool {
+	_, hasLabel := u.GetLabels()[providerLabel]
+	return !hasLabel
+}
+
 // Filters makes sure the disabled ingress resources are removed from the manifest.
 func Filters(ks *v1alpha1.KnativeServing) mf.Predicate {
 	var filters []mf.Predicate
 	if ks.Spec.Ingress == nil {
-		return mf.Any(istioFilter)
+		return istioFilter
```

Comment:
What is the difference between mf.Any(istioFilter) and istioFilter? They both return the `istioFilter`.

Reply:
They both return the **outcome** of `istioFilter`. This is just more lightweight as the Any wrapper is useless in this case.